### PR TITLE
Image gallery

### PR DESCRIPTION
**Pull request by HassanFdim**
Simple Image gallery using html and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,21 +7,23 @@
...
-      <a href="index.html"><</a>
-      <a href="index.html" class="active">1</a>
-      <a href="page2.html">2</a>
-      <a href="page3.html">3</a>
-      <a href="page4.html">4</a>
-      <a href="page5.html">5</a>
-      <a href="page2.html">></a>
+    <div class="gallery">
+      <a target="_blank" href="images/food 1.jpg">
+        <img src="images/food 1.jpg" alt="salad" />
```
Comment (Abdallah01win) [inline index.html]: Don't use spaces in file names 

`salad`

File: index.html [hunk 2]
```diff
@@ -7,21 +7,23 @@
...
+      </a>
+      <div class="description">salads</div>
+    </div>
+    <div class="gallery">
+      <a target="_blank" href="images/food 2.jpg">
+        <img src="images/food 2.jpg" alt="salad 2" />
```
Comment (Abdallah01win) [inline index.html]: same thing

File: index.html [hunk 3]
```diff
@@ -7,21 +7,23 @@
...
+      </a>
+      <div class="description">salads 2</div>
+    </div>
+    <div class="gallery">
+      <a target="_blank" href="images/food 3.jpg">
+        <img src="images/food 3.jpg" alt="salad 3" />
```
Comment (Abdallah01win) [inline index.html]: same thing

**Review by Abdallah01win — CHANGES_REQUESTED** [review]

**Review by HassanFdim — COMMENTED** [review]
I removed the spaces in the code

**Review by Abdallah01win — APPROVED** [review]

**Pull request merged**